tests/commons/j2/test_j2_functions.py

```diff
@@ -169,27 +169,32 @@ def test_string(self):
             type_name = pfx + short_name
             self.assertEqual(
                 xsd_fmt("Hello!", type_name),
-                "'Hello!'^^xsd:string",
+                "'\"Hello!\"'^^xsd:string",
```

Comment:
euh? this looks like we are introducing unwanted wrapping quotes now?

this test should totally not have been affected by this new implementation?